.github/workflows/velox_be.yml

```diff
@@ -131,6 +131,55 @@ jobs:
         run: |
           docker stop velox-backend-ubuntu2004-test-$GITHUB_RUN_ID || true
 
+  velox-backend-ubuntu2004-test-slow:
+    runs-on: velox-self-hosted
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup docker container
+        run: |
+          EXTRA_DOCKER_OPTIONS="--name velox-backend-ubuntu2004-test-slow-$GITHUB_RUN_ID -e NUM_THREADS=30 --detach" \
+          NON_INTERACTIVE=ON \
+          MOUNT_MAVEN_CACHE=OFF \
+          OS_IMAGE=ubuntu:20.04 \
+          tools/gluten-te/ubuntu/cbash.sh sleep 14400
+      - name: Setup maven cache
+        run: |
+          docker cp ~/.m2/repository velox-backend-ubuntu2004-test-slow-$GITHUB_RUN_ID:/root/.m2/
+      - name: Build Gluten Velox-backend third party
+        run: |
+          docker exec velox-backend-ubuntu2004-test-slow-$GITHUB_RUN_ID bash -c '
+          cd /opt/gluten/ep/build-arrow/src && \
+          ./get_arrow.sh  && \
+          ./build_arrow.sh --build_tests=ON --build_benchmarks=ON && \
```

Comment:
`--build_tests=ON --build_benchmarks=ON` is not needed

Reply:
done